.github/workflows/pipeline.yml

```diff
@@ -8,13 +8,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.5, 2.6]
+        ruby: [2.5, 2.6, 2.7.5, '3.0', 3.1]
```

Comment:
You can use `2.7` to make sure we're always testing the latest patch version of ruby 2.7